packages/react-async/src/tests/component.test.tsx

```diff
@@ -35,11 +35,11 @@ const mount = createMount<{hydrated?: boolean}>({
 });
 
 function ResolvedComponent({name = 'friend'}: {name?: string}) {
-  return <div>Hello, {name}!</div>;
+  return <>Hello, {name}!</>;
```

Comment:
Can we just return strings directly? I forget if that's allowed.

Reply:
This change requires additional type changes in the `load` argument in `createAsyncComponent`.

```
Type 'Promise<({ name }: { name?: string | undefined; }) => string>' is not assignable to type 'Promise<Import<ComponentType<{ name?: string | undefined; }>>>'.ts(2322)
```